src/lang/it.rs

```diff
@@ -28,7 +28,7 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
         ("Enable File Transfer", "Abilita trasferimento file"),
         ("Enable TCP Tunneling", "Abilita tunnel TCP"),
         ("IP Whitelisting", "IP autorizzati"),
-        ("ID/Relay Server", "Server ID/Relay"),
+        ("ID/Relay Server", "ID server/Relay"),
```

Comment:
Really? We need another Italian to confirm this.

Reply:
"Server ID" is English.
Or your mention could be Server ID/Relay ID?
The ID is it for both Server and Relay?

In English the workflow for the period is different then Italian.
Then some adjective could be get wrong if is not clear English meaning.